### ISSUE TITLE
Does it support retransformer and removeTransformer?

### ISSUE THREAD

**Issue by xiaoshao** [brutusin/instrumentation#1]

**Comment by idelvall** [comment]
Yes, it should. The library is designed to support multiple transformers registered. Each one of them would be represented by the `callbackId` value.
The code in 
https://github.com/brutusin/instrumentation/blob/master/src/main/java/org/brutusin/instrumentation/Agent.java#L29 should give you some insights on how to do that.
Let me know if you need help.
Cheers

**Comment by idelvall** [comment]
BTW, your question made me rethink (it's been a while), and raised new questions (https://github.com/brutusin/instrumentation/issues/3)

**Comment by xiaoshao** [comment]
I checked the code, it  just add transform.  no retransformer and removeTransformer

**Comment by weipengd** [comment]
> _No description provided._

I need some guidance, which is directly specified -javaagent: instrumentation?
